extra/release_tool.py

```diff
@@ -362,6 +358,14 @@ def yml_components(self):
     "mender-qa": "MENDER_QA_REV",
 }
 
+class JenkinsParam:
+    type = None
+    value = None
```

Comment:
Why do you want these defaults?

Reply:
I just need something. Got better ones? :-)